packages/headings/LinkedHeading.jsx

```diff
@@ -37,15 +50,24 @@ export function LinkedHeading({
           data-test="cta"
           href={ctaLink}
           target={ctaNewWindow ? "_blank" : undefined}
-          className="su-flex su-no-underline hover:su-underline hover:su-text-digital-red dark:hover:su-text-dark-mode-red su-transition su-items-center md:su-items-end md:su-mb-8 lg:su-mb-12 su-text-black dark:su-text-white su-flex-nowrap su-gap-20 md:su-gap-13 su-align-baseline su-text-19"
+          className={cnb(
+            "su-group su-flex su-no-underline hocus:su-underline su-transition su-items-center md:su-items-end md:su-mb-8 lg:su-mb-12 su-flex-nowrap su-align-baseline su-gap-20 md:su-gap-13 su-text-19 su-underline-offset-4 dark:su-text-white",
+            isAlwaysLight
+              ? "su-text-white hocus:su-text-white dark:hocus:su-text-white su-decoration-2"
```

Comment:
I left the hocus color of the heading CTA link as white because dark mode red won't meet contrast requirements on the background image with overlay. We can discuss this more.